lib/vet360/contact_information/service.rb

```diff
@@ -102,6 +102,7 @@ def get_telephone_transaction_status(transaction_id)
       def get_person_transaction_status(transaction_id)
         with_monitoring do
           raw_response = perform(:get, "status/#{transaction_id}")
+          Vet360::Stats.increment_transaction_results(raw_response, 'init_vet360_id')
```

Comment:
The Breakers StatsD plugin does give us stats about HTTP methods for free. It breaks down by endpoint, service, http method, and success/failure. I'm not sure if that covers what you need here or not, but this might be duplicate of what's already built-in.